wyw-in-js.config.cjs

```diff
@@ -0,0 +1,30 @@
+// const prefixes = ['@raw-', '@use-'];
+// const duplicates = {};
+
+module.exports = {
+  classNameSlug: (hash, title) => {
+    return hash;
+    // let shouldCheckDuplicates = false;
+    // let className = hash;
+    //
+    // for (let i = 0; i < prefixes.length; i++) {
+    //   if (title.startsWith(prefixes[i])) {
+    //     className = title.replace(prefixes[i], prefixes[i] === '@label-' ? 'sendbird-label--color-' : '');
+    //     shouldCheckDuplicates = true;
+    //     break;
+    //   }
+    // }
+    //
+    // if (shouldCheckDuplicates) {
+    //   if (duplicates[className]) {
+    //     // print error message in shell with bold text..
+    //     console.log(`\n\x1b[1m\x1b[33m[ERROR] Duplicate className: ${className} for ${title}\x1b[0m\n`);
+    //     throw new Error(`Duplicate className: ${className}`);
+    //   } else {
+    //     duplicates[className] = true;
+    //   }
+    // }
+    //
+    // return className;
```

Comment:
could you explain why we need this please?

Reply:
It is simply the configuration of the library. 

https://wyw-in-js.dev/configuration